rdmo/questions/viewsets.py

```diff
@@ -88,6 +96,15 @@ def detail_export(self, request, pk=None):
         xml = CatalogRenderer().render([serializer.data])
         return XMLResponse(xml, name=self.get_object().key)
 
+    @action(detail=False)
+    def current_site(self, request):
```

Comment:
this action definition is optional for `current_site` is optional because Ive placed the filter inside the `index` action